scripts/scenarios/cloud8/cloud8-5nodes-compute-ha.yml

```diff
@@ -4,20 +4,19 @@ proposals:
   name: services
   attributes:
     stonith:
-      mode: libvirt
-      libvirt:
-        hypervisor_ip: ##hypervisor_ip##
+      mode: sbd
```

Comment:
This will work also for the mkcloud jobs, correct?

Reply:
I don't see why not. But if this is about updating existing scenario files, see my previous comment: I reverted that and created two new batch scenario files for this purpose.